examples/seq2seq/README.md

```diff
@@ -128,6 +128,7 @@ Here is an example command, but you can do whatever you want. Hopefully this wil
 You can see your wandb logs [here](https://app.wandb.ai/sshleifer/hf_xsum?workspace=user-)
 
 ### DistilBART
+![DBART](https://github.com/sshleifer/transformers_fork/raw/add-distilbart-pic/examples/seq2seq/distilbart_w_logos.png)
```

Comment:
I’ll make you a version that is suitable for README (like the one I used for PABEE). Also I think uploading the image to our own server is a good idea, like this: https://huggingface.co/front/thumbnails/microsoft.png

Reply:
I've sent @sshleifer on Slack